components/TARGET_PSA/TARGET_TFM/COMPONENT_SPE/secure_fw/spm/spm_db.h

```diff
@@ -62,13 +62,18 @@ struct spm_partition_desc_t {
 #endif
 #ifdef TFM_PSA_API
     struct tfm_thrd_ctx sp_thrd;
+    /*
+     * stack_bottom points to starting address of the partitions' stack plus the partitions' stack top.
+     */
+    uint32_t stack_bottom;
+    uint32_t stack_top;
 #endif
 };
 
 /* Macros to pick linker symbols and allow to form the partition data base */
 #define REGION(a, b, c) a##b##c
 #define REGION_NAME(a, b, c) REGION(a, b, c)
-#if (TFM_LVL == 1) && !defined(TFM_PSA_API)
+#if (TFM_LVL == 1)
```

Comment:
Please comment in our patch why this check is changed from the original.

Reply:
done